pfb-analysis/import/import_neighborhood.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 NB_INPUT_SRID="${NB_INPUT_SRID:-4326}"
-NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-4326}"
-NB_BOUNDARY_BUFFER="${NB_BOUNDARY_BUFFER:-0}"
+NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-2163}"
```

Comment:
MA state plane is 2249.  2163 is "US National Atlas Equal Area".  The problem with 4326 as an output SRID is it's lat/lon, so the default max_trip_distance of 3300, intended to be about 2 miles, becomes the whole world.  So I went looking for a projection in meters that could make sense as a baseline default.

Alternatives include:
- Different default projection
- Leave the default as 4326 and change the default max trip distance to a value in degrees (not sure whether the pgrouting algorithms would handle that properly, but a very quick google made me think they might)
- Set the output SRID to 4326 and the max trip distance to something in meters* and don't worry about it

I meant to ask: is there a reason you went with 3600 as a default when switching the default commands from feet to meters?  I had been using 3300, since it's just over 2 miles.